syntax = "proto3";
option go_package = "gen/service";

package service;

import "google/protobuf/timestamp.proto";

message ClientResponseInfo{
    google.protobuf.Timestamp request_time = 1;
    google.protobuf.Timestamp response_time = 2;
    string language = 3;
}

message HelloResponse{
    string message = 1;
    ClientResponseInfo response_info = 2;
}


message CalculatorResponse{
    float addition = 1;
    float subtraction = 2;
    float multiplication = 3;
    float division = 4;
    ClientResponseInfo response_info = 5;
}